Zend/zend_API.h

```diff
@@ -266,6 +266,8 @@ ZEND_API int zend_copy_parameters_array(int param_count, zval *argument_array);
 #define zend_parse_parameters_none_throw() \
 	(EXPECTED(ZEND_NUM_ARGS() == 0) ? SUCCESS : zend_wrong_parameters_none_exception())
 
+#define ZEND_PARSE_PARAMS_NONE() { if(ZEND_NUM_ARGS() != 0 && zend_wrong_parameters_none_error()) return; }
```

Comment:
There already is a `ZEND_PARSE_PARAMETERS_NONE` macro which does the same thing ;)

Reply:
I do need to study more C, I thought that one was different because of the `do/while` struct :man_facepalming:

Reply:
@carusogabriel Function-like macros with more than one statement are usually wrapped in do/while, see https://stackoverflow.com/questions/154136/why-use-apparently-meaningless-do-while-and-if-else-statements-in-macros.